pkg/core/resources/store/error.go

```diff
@@ -0,0 +1,61 @@
+package store
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/kumahq/kuma/pkg/core/resources/model"
+)
+
+var (
+	ErrInvalidOffset   = errors.New("invalid offset")
+	ErrIsAlreadyExists = errors.New("already exists")
+	ErrConflict        = errors.New("conflict")
+	ErrNotFound        = errors.New("not found")
+	ErrInvalid         = errors.New("invalid")
+	ErrBadRequest      = errors.New("bad request")
+)
+
+func IsInvalidOffset(err error) bool {
+	return err != nil && errors.Is(err, ErrInvalidOffset)
```

Comment:
I think you don't need the `err != nil` because that's the code for errors.Is:
```
func Is(err, target error) bool {
	if err == nil || target == nil {
		return err == target
	}

	isComparable := reflectlite.TypeOf(target).Comparable()
	return is(err, target, isComparable)
}
```